src/openms_gui/source/VISUAL/TOPPViewIdentificationViewBehavior.cpp

```diff
@@ -495,7 +495,11 @@ namespace OpenMS
 
     for (Size i = 0; i < frag_annotations.size(); ++i)
     {
-      if (frag_annotations[i].annotation.hasSubstring(String("[alpha|")) || frag_annotations[i].annotation.hasSubstring(String("[beta|")))
+      bool has_alpha = frag_annotations[i].annotation.hasSubstring(String("alpha|"));
+      bool has_beta = frag_annotations[i].annotation.hasSubstring(String("beta|"));
+      // if it has both, it is a complex fragment and more difficult to parse
+      // those are ignored for the coverage indicator for now
+      if ( has_alpha ^ has_beta )
```

Comment:
please use || or "or" instead of bitwise operator^

Reply:
note: the first ones are logical operators the last one is bitwise operator.

Reply:
I needed an XOR in this case. Should I write it as
`(has_alpha || has_beta ) && !(has_alpha && has_beta)`?

Reply:
Just use A != B ;)

Reply:
right, thanks :)